pnpm-lock.yaml

```diff
@@ -30390,12 +30405,12 @@ snapshots:
       '@fluidframework/container-definitions': 2.22.0
       '@fluidframework/core-interfaces': 2.22.0
       '@fluidframework/core-utils': 2.22.0
-      '@fluidframework/datastore': 2.22.0(debug@4.4.0)
+      '@fluidframework/datastore': 2.22.0
       '@fluidframework/datastore-definitions': 2.22.0
-      '@fluidframework/map': 2.22.0(debug@4.4.0)
+      '@fluidframework/map': 2.22.0
       '@fluidframework/register-collection': 2.22.0
       '@fluidframework/runtime-definitions': 2.22.0
-      '@fluidframework/runtime-utils': 2.22.0(debug@4.4.0)
+      '@fluidframework/runtime-utils': 2.22.0
```

Comment:
I still don't understand why sometimes pnpm will "mix" or "unmix" (like in this case) dependencies like this. A lot of dependencies are now saying they're happy with `debug@4.3.7` instead of `4.4.0`, which also results in a lot of almost-duplicates below (e.g. line 30450 for `@fluidframework/aqueduct@2.22.0` is redundant-ish with line 30470 for `@fluidframework/aqueduct@2.22.0(debug@4.4.0)`.

Reply:
Gone after merging latest main 🤷 .